jenkins/scripts/integration_test.sh

```diff
@@ -50,14 +50,16 @@ VM_TIMELABEL="${VM_TIMELABEL:-$(date '+%Y%m%d%H%M%S')}"
 TEST_EXECUTER_VM_NAME="${TEST_EXECUTER_VM_NAME:-ci-test-vm-${VM_TIMELABEL}}"
 TEST_EXECUTER_PORT_NAME="${TEST_EXECUTER_PORT_NAME:-${TEST_EXECUTER_VM_NAME}-int-port}"
 
+if [[ "${TESTS_FOR}" == "feature_tests"* ]]
+then
+  OS_REGION_NAME="Fra1"
+  OS_AUTH_URL="https://fra1.citycloud.com:5000"
+fi
+
 if [[ "${TESTS_FOR}" == "feature_tests"* || "${TESTS_FOR}" == "e2e_tests"* ]]
 then
-  if [ "${DISTRIBUTION}" == "ubuntu" ]
-  then
-    TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-8C-32GB}"
-  else
-    TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-8C-32GB-300GB}"
-  fi
+    # Setting this to 8C-32GB-400GB since other flavors are unavailable in Frankfurt region
+    TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-8C-32GB-400GB}"
```

Comment:
So, there will not be any volume being used in this region? we had `8C-32GB` for Ubuntu since we attach volume to it and `8C-32GB-300GB` for CentOS where volume is out of picture.

Reply:
There is no `8C-32GB` flavor in Frankfurt. Ubuntu does create the volume similar to what is done previously. Check the current runs.